public-docs/how-to-extend-product.md

```diff
@@ -0,0 +1,112 @@
+---
+title: How To: Extend the Product Schema
+---
+
+## Prerequisite Reading
+- [Catalogs](./developer-catalog.md)
+- [Understanding Plugins](./core-plugins-intro.md)
+
+## Overview
+
+As a developer customizing Reaction, you may find a need to add some custom property to products. You should avoid this if you can achieve your goals some other way, such as using `metafields`, tags, or a separate data store that references product IDs. But in some cases, extending products is the best way.
+
+Because products have a publishing flow and have variants, extending them requires many steps. In general, they are as follows:
+- Extend database schemas
+- Extend GraphQL schemas
+- Register your custom property as published, if it should be published to the catalog
+- Register a function that publishes your custom property, if it should be published to the catalog
+- Create a GraphQL mutation for setting your custom property value
+- Create a React component that allows an operator to set the custom property value, and wire it up to your mutation, or set your property in some other way
+
+### Extend database schemas
+
+To extend any database schema, you just need a file that is imported into server code. We recommend using a file named `dbschema.js` in your plugin's `server/no-meteor` folder.
```

Comment:
Where is `dbschema.js` supposed to be imported?
I guess some new way should be used instead of `/server/index.js`, as this file resides in the no-meteor folder?